Mjml.Net/MjmlOptions.cs

```diff
@@ -34,6 +38,12 @@ public sealed record MjmlOptions
             ["&reg;"] = "&#174;",
             ["&trade;"] = "&#8482;"
         };
+        private IReadOnlyDictionary<string, string> xmlEntities = DefaultXmlEntities;
+
+        static MjmlOptions()
+        {
+            DefaultParserContext = BuilContext(DefaultXmlEntities);
```

Comment:
A small typo here should be `BuildContext`

Reply:
Can change that.

Reply:
Done